src/index.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+// const { default: axios } = require("axios");
+
+const DEFAULT_CITY = 'Seattle';
+// Select the HTML Element the event will occur on
+const increaseTempElement = document.getElementById('increaseTempBtn');
+const decreaseTempElement = document.getElementById('decreaseTempBtn');
+const currentTempTag = document.getElementById('currentTemp');
+const landscape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const headerCityName = document.getElementById('headerCityName');
+const cityNameReset = document.getElementById('cityNameReset');
+const SkyDropDown = document.getElementById('SkyChoice');
+const realtimeTemp = document.getElementById('realTempBtn');
+
+const changingTempColorAndLandscape = () => {
+  let currentTemp = parseInt(currentTempTag.textContent);
+  if (currentTemp >= 80) {
+    currentTempTag.className = 'red';
+    landscape.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (currentTemp >= 70) {
+    currentTempTag.className = 'orange';
+    landscape.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (currentTemp >= 60) {
+    currentTempTag.className = 'yellow';
+    landscape.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (currentTemp >= 50) {
+    currentTempTag.className = 'green';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else {
+    currentTempTag.className = 'teal';
+    landscape.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
+
+const changingSky = () => {
+  if (SkyDropDown.value === 'sunny') {
+    sky.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️';
+  } else if (SkyDropDown.value === 'cloudy') {
```

Comment:
Like my comment above, how could this method be refactored so that it didn't have several if/else if statements?